.github/workflows/verify.yml

```diff
@@ -0,0 +1,65 @@
+name: Certora
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - certora/erc20
+      - certora/erc1155ext
```

Comment:
do we want to run that on every PR ?

Reply:
Maybe master branch push and PRs? This way direct pushes to master are checked along with all PRs.